ui/component/or-dashboard-builder/src/or-dashboard-settingspanel.ts

```diff
@@ -275,60 +302,327 @@ export class OrDashboardSettingsPanel extends LitElement {
+                            true,
+                            true) : html ``
+                        }
+
+                            <div>
+                                <or-mwc-input .type="${InputType.SELECT}" style="width: 100%;"
+                                              .options="${this._config.attributes}"
+                                              .value="${config.attribute}" label="${i18next.t('Attribute')}"
+                                              @or-mwc-input-changed="${(event: CustomEvent) => {
+                                                  this.handleAttributeSelect(event.detail.value);
+                                              }}"
+                                ></or-mwc-input>
+                            </div>
+
+                            <div style="padding: 0 10px 12px 10px;">
+                                <div class="switchMwcInputContainer">
+                                                                    <span>${i18next.t('Show labels')}</span>
+                                                                    <or-mwc-input .type="${InputType.SWITCH}" style="width: 70px;"
+                                                                                  .value="${config.showLabels}"
+                                                                                  @or-mwc-input-changed="${(event: CustomEvent) => {
+                                                                                      this._config.showLabels = event.detail.value;
+                                                                                      this.updateParent(new Map<string, any>([["config", this._config]]));
+                                                                                  }}"
+                                                                    ></or-mwc-input>
+                                                                </div>
+                                                                <div class="switchMwcInputContainer">
+                                    <span>${i18next.t('Show units')}</span>
+                                    <or-mwc-input .type="${InputType.SWITCH}" style="width: 70px;"
+                                                  .value="${config.showUnits}"
+                                                  .disabled="${!config.showLabels}"
+                                                  @or-mwc-input-changed="${(event: CustomEvent) => {
+                                                      this._config.showUnits = event.detail.value;
+                                                      this.updateParent(new Map<string, any>([["config", this._config]]));
+                                                  }}"
+                                    ></or-mwc-input>
+                                </div>
+
+                            </div>
+
+                    `: undefined}
+                </div>
+            `
```

Comment:
Might need to consider whether this would be a better fit in a seperate component. It is starting to get a bit more than a settings panel at this point 😄